src/tensorlake/cli/graphs.py

```diff
@@ -0,0 +1,73 @@
+import json
+
+import click
+from pydantic.json import pydantic_encoder
+from rich import print, print_json
+from rich.table import Table
+
+from tensorlake.cli._common import AuthContext, pass_auth
+
+
+@click.group()
+def graph():
+    """
+    Serverless Graph Management
+    """
+    pass
+
+
+@graph.command()
```

Comment:
Lets add subcommand help message like here: https://github.com/tensorlakeai/indexify/blob/main/indexify/src/indexify/cli/build_image.py#L10

Reply:
No need; for commands, [help can be specified in the docstring](https://click.palletsprojects.com/en/stable/documentation/) :-)